app/views/posts/index.html.erb

```diff
@@ -0,0 +1,5 @@
+
+    <header>Posts!</header>
+    <% @posts.each do |post|%>
+      <p><%= post.post_title %></p>
```

Comment:
It'd be helpful for the user if these post titles were links. Could use a link helper here.